src/app/tests/clients/binance/test_binance_client.py

```diff
@@ -0,0 +1,94 @@
+import requests
+import unittest
+from unittest.mock import patch, MagicMock
```

Comment:
Separating the "whole" and "partial" imports

```suggestion
import unittest

from unittest.mock import patch, MagicMock
```